modules/core/handler_modules.php

```diff
@@ -1103,6 +1103,9 @@ public function process() {
 
                 Hm_Msgs::add("Server saved");
                 $this->out('just_saved_credentials', true);
+                $this->out('imap_server_id', $this->imap_server_id);
```

Comment:
@Yannick243 we can avoid adding imap_folders_enabled in allowed output as we don't really need it in js with something like:

```
if ($this->module_is_supported('imap_folders')) {
    $this->out('imap_server_id', $this->imap_server_id);
    $this->out('imap_service_name', $provider);
}
```